### PR TITLE
feat(ui): Add footer to discover chart on release detail page

### PR DESCRIPTION
**Pull request by matejminar**
This PR adds a footer to discover chart on release detail page.
This chart is shown instead of health stats chart when no health data are available.
We also hid this chart behind a feature flag and added new props to the event chart to be able to reuse it.

image

I would like to move `app/views/events/eventsChart.jsx ` to  `app/components/chart` in later PR now that it's being reused.

### CODE REVIEW

File: src/sentry/static/sentry/app/views/events/eventsChart.jsx [hunk 1]
```diff
@@ -52,8 +52,8 @@ class EventsAreaChart extends React.Component {
 
   render() {
     const {
-      loading, // eslint-disable-line no-unused-vars
-      reloading, // eslint-disable-line no-unused-vars
+      loading: _loading,
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/views/events/eventsChart.jsx:55]: 👌

File: src/sentry/static/sentry/app/views/releasesV2/detail/overview/chart/discoverChartContainer.tsx [hunk 2]
```diff
@@ -23,7 +32,45 @@ type Props = {
...
+
+    try {
+      const totalEvents = await fetchTotalCount(
+        api,
+        organization.slug,
+        this.getEventView().getEventsAPIPayload(location)
+      );
+      this.setState({totalEvents});
+    } catch (err) {
+      Sentry.captureException(err);
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/views/releasesV2/detail/overview/chart/discoverChartContainer.tsx:70]: 👏

**Review by priscilawebdev — APPROVED** [review]
Great Job! Codewise everything looks good to me 👏

File: src/sentry/static/sentry/app/views/releasesV2/detail/overview/chart/discoverChartContainer.tsx (hunk 2 above)
Comment (billyvg) [reply src/sentry/static/sentry/app/views/releasesV2/detail/overview/chart/discoverChartContainer.tsx:70]: This is a bit redundant, Sentry will capture all unhandled rejections. We generally do this if we have our own error handling logic to add.

**Review by billyvg — APPROVED** [review]

**Pull request merged**